### DISCUSSION TITLE
Как на внешней форме открыть закрытую вкладку по условию?

### DISCUSSION THREAD

**Discussion by Re-Requeste** [totumonline/totum-ru-issues-and-discussions#549] [Как это сделать?]
Добрый день, не оставил изучение внешних форм, и добрался до интересного момента.
Если изначально закрыть секцию (Close), можно ли ее открыть по определенному условию?

Допустим, секция:  3.СКРЫТОЕ ПОЛЕ - закрыт
![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/dc86ea77-7d3e-453f-9d07-83c409e08d74)

 и хотелось бы ее открыть если нажать чекбокс:

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/137899742/41e47254-061d-43ee-830d-15b1b9889358)

Исследование показал следующее: 
**setFormSectionsFormat** - думал это подойдет, но нет , т.е. в статус код секции он не срабатывает,  _код, код действий, код форматирование_ он вообще не реагирует либо ругается что данной функции нет.

Не подскажите в какую сторону копать?

**Comment by ttm-alexeypolunin** [comment]
А нужно именно Секцию? Тк если там видимостью конкретного поля управлять, то там все проще.

**Reply by Re-Requeste** [reply]
Да желательно б секцию, с полями разобрался,там через [setFormat] hide form (https://ru.docs.totum.online/functions#fn-setFormat)

Из-за setFormat я подумал что setFormSectionsFormat это решение, но видимо ошибся. Хотя там регулировка есть
